app/templates/editor.static.jade

```diff
@@ -34,6 +34,22 @@ section#container-edit.container-fluid.full-height
 
                 .controls-entry
                     .row
+                        h3 Banner
+                    .row
+                        from(role='form')#banner
+                            .radio-inline
+                                input(type='radio', name='radio-banner', checked='checked')#banner-none
+                                label(for='banner-none')
+                                    span
+                                    span None
+                            .radio-inline
+                                input(type='radio', name='radio-banner')#banner-beta
+                                label(for='banner-beta')
+                                    span
+                                    span Beta
+
+                .controls-entry
+                    .row
```

Comment:
This we should probably hide behind some collapsible UI since there will have to be more options to configure the banner.